src/legacy/index.ts

```diff
@@ -78,7 +78,7 @@ async function buildGraph(
     eventLoopSpinner: EventLoopSpinner,
     isRoot = false): Promise<string> {
 
-  const getNodeId = (name: string, version: string | undefined, hashId: string) =>
+  const getNodeId = (name: string, version: string|null|undefined, hashId: string) =>
```

Comment:
should better have spaces around the `|` - that's how it appears in the docs: https://www.typescriptlang.org/docs/handbook/advanced-types.html#union-types.